tests/xharness/Jenkins.cs

```diff
@@ -40,6 +40,11 @@ public class Jenkins
 		public bool IncludeXtro;
 		public bool IncludeCecil;
 		public bool IncludeDocs;
+		public bool IncludeNewBCL;
```

Comment:
I prefer Sebastien's naming (BCL-nunit and BCL-xunit)

Reply:
yeah, no problem, as I said, I'm terrible at naming :)